build_projects/update-dependencies/UpdateFilesTargets.cs

```diff
@@ -92,6 +92,7 @@ public static BuildTargetResult ReplaceProjectJson(BuildTargetContext c)
             {
                 Path.Combine(Dirs.PkgProjects, "Microsoft.NETCore.App", "project.json.template"),
                 Path.Combine(Dirs.PkgProjects, "Microsoft.NETCore.UniversalWindowsPlatform", "project.json.template"),
+                Path.Combine(Dirs.PkgProjects, "Microsoft.NET.CoreRuntime", "project.json.template"),
```

Comment:
FYI - this file is removed with my latest PR.

You should be able to apply this change here:

https://github.com/dotnet/core-setup/pull/2125/files#diff-2a53f5c279da5b2626bb2fb53cf9d005R175